src/gflags_completions.cc

```diff
@@ -325,12 +325,21 @@ static void CanonicalizeCursorWordAndSearchOptions(
     }
     break;
   }
-
-  switch (found_question_marks) {  // all fallthroughs
-    case 3: options->flag_description_substring_search = true;
-    case 2: options->flag_location_substring_search = true;
-    case 1: options->flag_name_substring_search = true;
-  };
+  
+  if (found_question_marks == 3) {
```

Comment:
Thanks. This may be more readable indeed, but what do you think about:
```
if (found_question_marks > 2) options->flag_description_substring_search = true;
if (found_question_marks > 1) options->flag_location_substring_search = true;
if (found_question_marks > 0) options->flag_name_substring_search = true;
```

Reply:
Looks nice! Thanks